src/client_side_reply.cc

```diff
@@ -1562,17 +1561,14 @@ clientReplyContext::buildReplyHeader()
         Auth::UserRequest::AddReplyAuthHeader(reply, request->auth_user_request, request, http->flags.accel, 0);
 #endif
 
-    /* Append X-Cache */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE, "%s from %s",
-                      is_hit ? "HIT" : "MISS", getMyHostname());
-
-#if USE_CACHE_DIGESTS
-    /* Append X-Cache-Lookup: -- temporary hack, to be removed @?@ @?@ */
-    httpHeaderPutStrf(hdr, Http::HdrType::X_CACHE_LOOKUP, "%s from %s:%d",
-                      lookup_type ? lookup_type : "NONE",
-                      getMyHostname(), getMyPort());
-
-#endif
+    SBuf cacheStatus(uniqueHostname());
+    cacheStatus.append(is_hit ? ";hit" : ";fwd=miss");
```

Comment:
S-F parameter serialization [algorithm](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-19#section-4.1.1.2) implies no SP after `;`. Our implementation follows that suggestion. AFAICT, _all_ Cache-Status draft examples do not follow that suggestion -- they use `;` SP sequences for all parameters. It could be simply for readability/presentation purposes.

I think we should follow the S-F draft here rather than examples (so no changes are necessary), but I wanted to bring your attention to this mismatch in case you disagree.

Reply:
Noticed.